contracts/deploy/02-inbox/02-arb-to-gnosis-inbox.ts

```diff
@@ -1,56 +1,78 @@
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { DeployFunction } from "hardhat-deploy/types";
 import getContractAddress from "../../deploy-helpers/getContractAddress";
+import { ethers } from "hardhat";
 
 enum SenderChains {
-  ARBITRUM = 42161,
-  ARBITRUM_SEPOLIA = 421614,
+  GNOSIS_MAINNET = 100,
+  GNOSIS_CHIADO = 10200,
   HARDHAT = 31337,
 }
+
 const paramsByChainId = {
-  ARBITRUM: {
-    epochPeriod: 3600, // 1 hours
-  },
-  ARBITRUM_SEPOLIA: {
-    epochPeriod: 3600, // 1 hours
-  },
+  GNOSIS_MAINNET: {},
+  GNOSIS_CHIADO: {},
   HARDHAT: {
-    epochPeriod: 600, // 10 minutes
+    epochPeriod: 600, // 10 min
+    routerAddress: ethers.constants.AddressZero,
   },
 };
 
-// TODO: use deterministic deployments
 const deployInbox: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
   const { ethers, deployments, getNamedAccounts, getChainId, config } = hre;
   const { deploy } = deployments;
-  const chainId = Number(await getChainId());
-  const { providers } = ethers;
 
-  const deployer = (await getNamedAccounts()).deployer;
-  console.log("deployer: %s", deployer);
+  // fallback to hardhat node signers on local network
+  const deployer = (await getNamedAccounts()).deployer ?? (await hre.ethers.getSigners())[0].address;
```

Comment:
It will be more verbose (maybe more readable, on the other side), but the getNamedAccounts and getChanId operations can be run concurrently with `Promise.all`.

Reply:
we're using this script to deploy test contracts in HRE, so we don't really need to sweat the performance too much. Do you think it's still worth making the code a bit wordier for the sake of readability? Or should we keep it simple?

Reply:
though being a test script ,performance might make sense here so as the project and test suite grows it wouldn't block the CI.

Reply:
Usually, the rule of thumb for me is that behaviors can be concurrent should be running concurrently by default.